build/config.props

```diff
@@ -38,9 +38,9 @@
   <PropertyGroup>
     <NewtonsoftJsonVersionCore Condition="$(NewtonsoftJsonPackageVersion) == ''">9.0.1</NewtonsoftJsonVersionCore>
     <NewtonsoftJsonVersionCore Condition="$(NewtonsoftJsonPackageVersion) != ''">$(NewtonsoftJsonPackageVersion)</NewtonsoftJsonVersionCore>
-    <XunitVersion>2.3.1</XunitVersion>
+    <XunitVersion>2.4.1</XunitVersion>
```

Comment:
Why do we need to update the xunit versions? 

Note that there's a bootstrapping step that downloads the actual runner in .nuget/packages.config, so we'd need to change that too.

Reply:
changing the version in `.nuget/packages.config` will probably need changes to `scripts/funcTests/runFuncTests.sh` and `build/build.proj`

Reply:
I was wondering where the hidden dependencies to xunit version were... 😄 
I figured it doesn't harm to upgrade whilst at it.

Reply:
Fixed in https://github.com/NuGet/NuGet.Client/pull/2973/commits/923427c7a5d204624c5c5aafb3fe501916765e00.

No changes were needed in `build/build.proj` AFAICT.

Reply:
ok, different file :)

https://github.com/NuGet/NuGet.Client/blob/ea1a5fb2afe5149389bea7befb7c8a076f457274/build/common.project.props#L30

Reply:
Thanks :) https://github.com/NuGet/NuGet.Client/pull/2973/commits/47eedb3f076976d4653a33b92d3ea26f4959138f